examples/launch_pasha_nasbench201_demo.py

```diff
@@ -0,0 +1,424 @@
+# PASHA: Efficient HPO with Progressive Resource Allocation
```

Comment:
Hello Ondre,

thanks a lot for this great example.

My feeling is it is a bit too long for what we normally have in the examples folder. Also, it would be a bit wasted there, because it is almost more like a tutorial.

If others agree, and you'd be up for it, one idea would be to have this in docs/tutorials/pasha.

It would include this complete script here, and then a markdown with code snippets, where the user can learn about PASHA.

What do you think?

Reply:
I was thinking to suggest this also as it is indeed a bit long for examples. Either way works for me.

Reply:
I think it is a good idea to create a tutorial out of it - originally it was a Jupyter notebook, which has a similar style as a tutorial.

Reply:
Hi Ondre, could you please rather move it to docs/tutorials/pasha/? The folder basics is for a specific tutorial

Reply:
sure, I've just moved it there